src/output.rs

```diff
@@ -47,25 +49,32 @@ pub struct MidiOutput {
 impl MidiOutput {
     /// Update the available output ports.
     pub fn refresh_ports(&self) {
-        self.sender.send(Message::RefreshPorts).unwrap();
+        self.sender
+            .send(Message::RefreshPorts)
+            .expect(REFRESH_PORTS_ERROR_MSG);
     }
 
     /// Connect to the given `port`.
     pub fn connect(&self, port: MidiOutputPort) {
-        self.sender.send(Message::ConnectToPort(port)).unwrap();
+        self.sender
+            .send(Message::ConnectToPort(port))
+            .expect(CONNECT_TO_PORT_ERROR_MSG);
     }
 
     /// Disconnect from the current output port.
     pub fn disconnect(&self) {
-        self.sender.send(Message::DisconnectFromPort).unwrap();
+        self.sender
+            .send(Message::DisconnectFromPort)
+            .expect(DISCONNECT_FROM_PORT_ERROR_MSG);
     }
 
     /// Send a midi message.
     pub fn send(&self, msg: MidiMessage) {
-        self.sender.send(Message::Midi(msg)).unwrap();
+        self.sender.send(Message::Midi(msg)).expect("Send MIDI");
```

Comment:
Forgot to update this error message. Probably should be "Couldn't send MIDI message".